drupal-7/pipeline.yml

```diff
@@ -0,0 +1,50 @@
+jobs:
+- name: deploy-drupal-7
+  plan:
+  - aggregate: *plan-gets
```

Comment:
This reference doesn't have a corresponding anchor, so it's not valid.

Reply:
Thanks! I hadn't meant to include that particular file; I've removed it now and will finish it up in a different branch.